src/index.ts

```diff
@@ -1381,6 +1381,38 @@ class Remarkable implements RemarkableApi {
     return await this.bulkMove(hashes, "trash", refresh);
   }
 
+
+  /** permanent delete many hashes */
+  async bulkPurge(
```

Comment:
for type script support, you'll need to also add this definition to the interface declared earlier.